src/components/common/TmModalTutorial.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div
+    v-if="show"
+    v-focus-last
+    class="tm-modal-tutorial"
```

Comment:
no `tm` required. this is from Tendermint days. 😄

Reply:
All anachronic references removed